modules/integration/tests-integration/bpmn/src/test/java/org/wso2/bps/integration/tests/bpmn/BPMNCappDeploymentTestCase.java

```diff
@@ -29,20 +29,22 @@
 import java.io.File;
 import java.net.URL;
 import java.util.Calendar;
-import java.util.concurrent.TimeUnit;
 
 import org.wso2.carbon.integration.common.admin.client.ApplicationAdminClient;
 import org.wso2.bps.integration.common.utils.BPSMasterTest;
 
 import javax.activation.DataHandler;
 
-
+/**
+ * Deploying a cAPP file which includes bpmn artifacts
+ */
 public class BPMNCappDeploymentTestCase extends BPSMasterTest {
 
     private static final Log log = LogFactory.getLog(BPMNCappDeploymentTestCase.class);
-    String carFileName = "TestPro_1.0.0";
-    private boolean BPMNProcessExists = false;
     private final int MAX_TIME = 120000;
+    String carFileName = "TestPro_1.0.0";
+    String extension = ".car";
+    private boolean bpmnProcessExists = false;
     private boolean isCarFileUploaded = false;
     private ApplicationAdminClient applicationAdminClient;
```

Comment:
Added ".car" as a constant in BPMNTestConstants class and removed extension variable from BPMNCappDeploymentTestCase. Updated with commit ec83b77